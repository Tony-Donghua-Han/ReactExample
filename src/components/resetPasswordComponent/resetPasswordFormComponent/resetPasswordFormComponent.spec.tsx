import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import ResetPasswordFormComponent from "./resetPasswordFormComponent";

describe("ResetPasswordFormComponent", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<ResetPasswordFormComponent onSubmit={(): void => {}} />).dive();
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
