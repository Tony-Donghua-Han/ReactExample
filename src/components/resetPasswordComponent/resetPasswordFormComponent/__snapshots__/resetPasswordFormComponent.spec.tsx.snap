// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordFormComponent should render without crushing 1`] = `
"<ContextProvider value={{...}}>
  <Form className=\\"resetPasswordForm\\" id=\\"resetPasswordForm\\">
    <FormikConnect(FieldInner) type=\\"password\\" name=\\"newPassword\\" className=\\"inputBox\\" placeHolder=\\"New Password\\" />
    <FormikConnect(ErrorMessageImpl) name=\\"newPassword\\" className=\\"alert\\" component=\\"div\\" />
    <FormikConnect(FieldInner) type=\\"password\\" name=\\"confirmationPassword\\" className=\\"inputBox\\" placeHolder=\\"Confirm New Password\\" />
    <FormikConnect(ErrorMessageImpl) name=\\"confirmationPassword\\" className=\\"alert\\" component=\\"div\\" />
    <button type=\\"submit\\" disabled={false} className=\\"submitButton\\">
      Change Password
    </button>
  </Form>
</ContextProvider>"
`;
